src/docs/development/ui/layout/building-adaptive-apps.md

```diff
@@ -300,15 +297,14 @@ depended on a global value.
 
 The previous example could be rewritten using `LayoutBuilder`:
 
-<!--skip-->
+<?code-excerpt "lib/widgets/extra_widget_excerpts.dart (LayoutBuilder)"?>
 ```dart
-Widget foo = LayoutBuilder(builder: (_, constraints, __){
- bool useVerticalLayout = constraints.maxWidth < 400.0;
- return Flex(
-     children: [...],
-     direction: useVerticalLayout ?
-     Axis.vertical : Axis.horizontal
- );
+Widget foo = LayoutBuilder(
+    builder: (BuildContext context, BoxConstraints constraints) {
+  bool useVerticalLayout = constraints.maxWidth < 400.0;
+  return Flex(
+      children: [Text("Hello"), Text("World")],
+      direction: useVerticalLayout ? Axis.vertical : Axis.horizontal);
```

Comment:
Could we have a trailing comma here to make it prettier? Same as below codes.

Reply:
Hi @AlexV525! Thanks for catching that! Added in the trailing comma.